app/reducers/lnd.js

```diff
@@ -90,8 +90,9 @@ export const lndSyncStatus = (event, status) => async (dispatch, getState) => {
 // Connected to Lightning gRPC interface (lnd wallet is connected and unlocked)
 export const lightningGrpcActive = (event, lndConfig) => async dispatch => {
   dispatch({ type: SET_LIGHTNING_WALLET_ACTIVE })
-
-  // Once we we have established a connection, save the wallet settings.
+  // Once we we have established a connection, save the wallet settings
+  // after connection was successfully established. This is especially important
+  // for the first connection to be sure settings are correct  if (lndConfig.id !== 'tmp') {
```

Comment:
I think you can remove `if (lndConfig.id !== 'tmp') {` from this comment. It doesn't seem relevant. That condition just ensures that we are excluding the special `tmp` wallet (the one that we use to generate new seeds)

Reply:
Right. That's probably a merge artifact.